media_daemon/auto_skills.py

```diff
@@ -11,18 +11,23 @@ def make_skill(): # pylint-enable
     os.chdir('{}'.format(x[1]))
     USB = name_of_usb[0]
     mp3 = [file for file in glob("*.mp3")]
```

Comment:
Instead of mp3, I think we should also include other common audio formats, for example iTunes Store uses .aac.
.wav and .flac should be considered as well,

Reply:
true but if this can only be solved with more coding/changes than a simple condition then we should shift more media types in another issue. So we can move on quickly for now.